recipe/build.sh

```diff
@@ -54,6 +54,16 @@ if [[ "$target_platform" == "linux-aarch64" || "$target_platform" == "osx-arm64"
   TEST_CMD=""
 fi
 
+if [[ "$target_platform" == "osx-arm64" ]]; then
+  ARCH_OPTS_SINGLE="--enable-neon"
+  ARCH_OPTS_DOUBLE="--enable-neon"
+  ARCH_OPTS_LONG_DOUBLE="--enable-long-double"
```

Comment:
osx-arm64 supports `long double` but it's the same 64-bit width as `double`.  Does that matter?

Reply:
I guess so. Let's disable that